### PR TITLE
Rating import

### PR DESCRIPTION
**Pull request by srivig**

### CODE REVIEW

**Comment by srivig** [comment]
@chuenlok  .. thanks henry...

File: db/migrate_cards/20150119162522_add_sourcefile_type.rb [hunk 1]
```diff
@@ -7,8 +7,12 @@ def up
     else
       Card.create! :name=>'Source file', :type_id=>Card::CardtypeID, :codename=>'source_file'
     end
+<<<<<<< HEAD
+    Card.create! :name=>'Metric', :type_id=>Card::CardtypeID, :codename=>'metric'   
+=======
    
     #Card.create! :name=>'Metric', :type_id=>Card::CardtypeID, :codename=>'metric'
+>>>>>>> wagn-wikirate/master
```
Comment (ethn) [inline db/migrate_cards/20150119162522_add_sourcefile_type.rb:15]: I'll fix this conflict.

File: db/migrate_cards/data/metric_migration.json [hunk 2]
```diff
@@ -0,0 +1 @@
...emoveAttr('checked');\r\n          slot.find(\".import_table\").find(\"tr\").each(function() {\r\n            $this = $(this);\r\n            var match = $this.find(\"td:nth-child(4)\").html();\r\n            if(match==\"exact\")\r\n                    $this.find(\"td:nth-child(1)\").find(\"input:checkbox\").prop(\"checked\",true);\r\n            \r\n          });\r\n  }\r\n});\r\n});"},{"name":"*all+*script","type":"Pointer","content":"[[script: jquery]]\n[[script: tinymce]]\n[[script: slot]]\n[[script: card menu]]\n[[script: jquery helper]]\n[[wikirate coffee]]\n[[chosen proto script]]\n[[chosen script]]\n[[claim chosen]]\n[[claim clipboard script]]\n[[zeroclipboard]]\n[[tooltip script]]\n[[modal window script]]\n[[user profile tab script]]\n[[readmore_script]]\n[[page preview button link]]\n[[activate readmore script]]\n[[suggested source script]]\n[[empty tab content script]]\n[[source claim list script]]\n[[claim citation script]]\n[[script: filter search]]\n[[newPageJs]]\n[[newClaimJs]]\n[[homepage carousel init]]\n[[top bar shadow script]]\n[[slick js]]\n[[homepage ie8 handling script]]\n[[slick nav js]]\n[[mobile navigation js]]\n[[script: ace]]\n[[import page script]]"}]}}
```
Comment (ethn) [inline db/migrate_cards/data/metric_migration.json:1]: did the json come out in this one-line format from the export?  generally nice if this is readable...

Comment (chuenlok) [inline db/migrate_cards/data/metric_migration.json:1]: it is one-line format from the export now. let me try to make a pretty print on the json

**Pull request merged**